input.json

```diff
@@ -2,4 +2,4 @@
     "command": "InitUser",
     "username": "Alice",
     "password": "12345"
-}
```

Comment:
I'm not sure we want this file here, since each call to this filesystem will have to generate its own json file

Reply:
do we have other format to put the input command except json file and shell command line? I also don't want to pass in a file but I am not sure what other methods we have to pass the input command

Reply:
I think passing in a file is fine, but I think having an `input.json` file in the repo itself is probably not necessary (that is, we should generate it each time we want to run the `api.go` file)